powersimdata/design/investment/create_mapping_files.py

```diff
@@ -223,7 +223,7 @@ def write_poly_shapefile():
     """
     fiona = _check_import("fiona")
     shapely_geometry = _check_import("shapely.geometry")
-    Polygon = shapely_geometry.Polygon
+    polygon = shapely_geometry.Polygon
```

Comment:
Should we keep this as `Polygon` and `#noqa` the error? What we're really doing is importing a class, which I think we should keep uppercase if possible.

Reply:
Sure, I think it makes sense in this case.